src/yio-model/mediaplayer/tvchannelmodel_mediaplayer.h

```diff
@@ -0,0 +1,135 @@
+/******************************************************************************
+ *
+ * Copyright (C) 2020 Michael Lörcher <MichaelLoecher@web.de>
+ *
+ * This file is part of the YIO-Remote software project.
+ *
+ * YIO-Remote software is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * YIO-Remote software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with YIO-Remote software. If not, see <https://www.gnu.org/licenses/>.
+ *
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ *****************************************************************************/
+
+#pragma once
+
+#include <QAbstractListModel>
+#include <QObject>
+#include <QVariant>
+#include <QtDebug>
+
+class tvchannelModelItem {
+ public:
+    tvchannelModelItem(const QString& key, const QString& time, const QString& title,
+                       const QString& subtitle, const QString& type,
+              const QString& imageUrl, const QVariant& commands)
+        : m_key(key), m_time(time), m_title(title), m_subtitle(subtitle),
+          m_type(type), m_imageUrl(imageUrl), m_commands(commands) {}
+
+    QString  item_key() const { return m_key; }
+    QString  item_time() const { return m_time; }
+    QString  item_title() const { return m_title; }
+    QString  item_subtitle() const { return m_subtitle; }
+    QString  item_type() const { return m_type; }
+    QString  item_imageUrl() const { return m_imageUrl; }
+    QVariant item_commands() const { return m_commands; }
```

Comment:
Please avoid underscores in method names: `itemKey()`, `itemTime()`, etc

Reply:
Done